src/backend/TrafficCourts/Citizen.Service/Models/Disputes/SearchDisputeResult.cs

```diff
@@ -16,6 +16,9 @@ public class SearchDisputeResult
 
         [JsonPropertyName("hearing_type")]
         public JJDisputeHearingType? HearingType { get; set; }
+        
+        [JsonPropertyName("is_email_verified")]
+        public bool? IsEmailVerified { get; set; }
```

Comment:
So this will be a part of SearchDisputeResult not NoticeOfDispute.

Reply:
Yes, this is part of the small object returned by the workflow service when finding a dispute.